tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -138,6 +138,49 @@ steps:
                   exit 1
                 fi
             else
+              packageVersion=$(grep -oP 'npm notice version:\s+\K[^\s]+' publish_log)
```

Comment:
Does this need to be a bash script? You're making requests, doing a lot of logic that is not easy to test outside of CI. How could this be changed so it can be run and tested locally?